testdata/test/test_for.py

```diff
@@ -31,16 +32,18 @@ def main():
         k += 1
 
         if i == 15:
-            print("1 for break in");break
+            print("1 for break in")
+            break
 
         if i == 10:
-            print("1 for continue in");continue
```

Comment:
yes

Reply:
@carltraveler I don't quite understand the benefit of putting it on one line, since doing it on multiple lines works too. It also violates PEP8 E702 (https://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes). Can you explain why to put them on a single line?

Reply:
yes. you are right. but this is test code. I just try to test wether write in online or any other strange code style will get any bug or not. in boa if multi satement write in online will get bug